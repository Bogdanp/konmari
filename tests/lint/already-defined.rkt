#lang racket/base

(define a 42)
(define a 43)
(define (a x) x)
(define ((a x) y) (+ x y))
(define (((a x) y) z) (+ x y z))
