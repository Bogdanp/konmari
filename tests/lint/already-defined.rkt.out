/already-defined.rkt:4:9:error:identifier 'a' is already defined
/already-defined.rkt:5:10:error:identifier 'a' is already defined
/already-defined.rkt:6:11:error:identifier 'a' is already defined
/already-defined.rkt:7:12:error:identifier 'a' is already defined
/already-defined.rkt:3:9:warning:identifier 'a' is never used
